cordapp/contracts/src/main/java/com/newamerica/states/FundState.java

```diff
@@ -0,0 +1,95 @@
+package com.newamerica.states;
+
+import com.template.contracts.TemplateContract;
+import net.corda.core.contracts.BelongsToContract;
+import net.corda.core.contracts.LinearState;
+import net.corda.core.contracts.UniqueIdentifier;
+import net.corda.core.identity.AbstractParty;
+import net.corda.core.identity.Party;
+import net.corda.core.serialization.ConstructorForDeserialization;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Currency;
+import java.util.List;
+
+/**
+ * A FundState is an on-ledger representation of off-ledger recovered assets.
+ *
+ *  originCountry - the source country which will be sending the funds (US).
+ *  targetCountry - the target country which will be receiving the funds (Catan).
+ *  amount - the original value the fund held when it was issued.
+ *  balance - the current value that exists in the fund.
+ *  datetime - the day/time the fund was issued.
+ *  maxWithdrawalAmount - a term agreed upon in an off-leger bi-lateral agreement.
+ *  currency - the globally recognized currency for the fund balance and amount.
+ *  status - current stage of the fundState's lifecycle in Corda (can be ISSUED or PAID)
+ */
+
+@BelongsToContract(TemplateContract.class)
+public class FundState implements LinearState {
+    public final Party originCountry;
+    public final Party targetCountry;
+    public final double amount;
+    public final double balance;
+    public final LocalDate datetime;
```

Comment:
The LocalDate class in the Java 8 date time API represents a local date which is a date without time zone information. A local date could be a birthdate or official holiday etc. which relates to a specific day of the year - not the precise time of that day (the moment the day starts). We should probably use ZonedDateTime or equivalent that supports Time zone

Reply:
Applied an enum.